### PR TITLE
Add an open-loop test using Poisson at 20-70% of peak

### PR DESCRIPTION
**Pull request by vjpai**
Determine loaded latency. This depends on #5201 . The unique commit in this PR is 1a06082.

### CODE REVIEW

**Comment by vjpai** [comment]
Ready for a look.

File: test/cpp/qps/qps-sweep.sh [hunk 1]
```diff
@@ -71,7 +71,15 @@ for secure in true false; do
...
     --num_servers=1 --num_clients=0
 
-  # Scenario 3: Latency at near-peak load (TBD)
+  # Scenario 3: Latency at near-peak load (all clients equally loaded)
+  "$bins"/opt/qps_driver --rpc_type=STREAMING --client_type=ASYNC_CLIENT \
+    --server_type=ASYNC_GENERIC_SERVER --outstanding_rpcs_per_channel=100 \
+    --client_channels=64 --bbuf_req_size=0 --bbuf_resp_size=0 \
+    --async_client_threads=0 --async_server_threads=0 --secure_test=$secure \
+    --num_servers=1 --num_clients=0 --poisson_load=`awk '$5 == "QPS:" \
+      {print int(0.7 * $6); exit}' /tmp/qps-test.$$`
```
Comment (dgquintas) [inline test/cpp/qps/qps-sweep.sh]: perhaps make a variable hold the 0.7 factor?

Comment (dgquintas) [inline test/cpp/qps/qps-sweep.sh]: the name of that potential variable would also help clarify what the role of that factor is.

Comment (vjpai) [inline test/cpp/qps/qps-sweep.sh]: Done, also made it a variable variable

**Comment by vjpai** [comment]
Shell variables, shell variables everywhere

**Comment by vjpai** [comment]
Thanks for the comments; PTAL.

File: test/cpp/qps/qps-sweep.sh [hunk 2]
```diff
@@ -40,6 +40,8 @@ bins=`find . .. ../.. ../../.. -name bins | head -1`
 set -x
 
 big=65536
+wide=64
```
Comment (dgquintas) [inline test/cpp/qps/qps-sweep.sh]: docstrings for these three variables plz :)

Comment (vjpai) [inline test/cpp/qps/qps-sweep.sh]: Did it, and also for half.

**Comment by dgquintas** [comment]
LGTM, feel free to merge

**Pull request merged**